x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/services/cohere/CohereService.java

```diff
@@ -174,6 +176,6 @@ private CohereEmbeddingsModel updateModelWithEmbeddingDetails(CohereEmbeddingsMo
 
     @Override
     public TransportVersion getMinimalSupportedVersion() {
-        return TransportVersions.ML_INFERENCE_COHERE_EMBEDDINGS_ADDED;
+        return TransportVersions.ML_INFERENCE_REQUEST_INPUT_TYPE_UNSPECIFIED_ADDED;
```

Comment:
Bumping the cohere service min version just in case.

Reply:
Yeah, it means that if a cluster has nodes with different versions and one of the nodes is on `ML_INFERENCE_REQUEST_INPUT_TYPE_UNSPECIFIED_ADDED` and a user attempts to create the cohere entity, it will fail until all the nodes are on the version `ML_INFERENCE_REQUEST_INPUT_TYPE_UNSPECIFIED_ADDED`. Essentially users will have to wait until an upgrade process is complete before creating this inference entity.